pkg/api/routes.go

```diff
@@ -27,6 +27,7 @@ import (
 	artifactspec "github.com/oras-project/artifacts-spec/specs-go/v1"
 	zerr "zotregistry.io/zot/errors"
 	"zotregistry.io/zot/pkg/api/constants"
+	debugGQLPlayground "zotregistry.io/zot/pkg/debug/gqlplayground"
```

Comment:
just call this "gqlPlayground".
"debug" is how we build this.

Reply:
Renamed